### ISSUE TITLE
Offer an option for pull quotes in articles

### ISSUE THREAD

**Issue by biofilo** [crimethinc/website#458]
As it stands, we can do ordinary quotations in articles by preceding each paragraph with a >. But there is not option to make a line bigger and bolder than the rest of the text. I've been using the current quotation model to insert quotations into the middle of an article, but I'd like a way to do pull quotes.

**Comment by veganstraightedge** [comment]
`> ` is the pull quote syntax.

Are you just asking for a visual style to make some pull quotes bigger and bolder?

**Comment by astronaut-wannabe** [comment]
pull quotes aren't supported in markdown, but here are a couple of techniques I found to do simple pull quotes with CSS:

- http://miekd.com/articles/pull-quotes-with-html5-and-css/
- https://codepen.io/aarongustafson/pen/zxKdh

**Comment by veganstraightedge** [comment]
I'll wait for @biofilo's clarification, but I don't think he means a *pull quote*. I _think_ he means a block with bigger/bolder styles than the default block quote style. If that's the case, we can do that with just CSS and a special class for authors to add to a block quote item.

***

Kramdown (our Markdown parser) provides a special syntax for adding css ids and classes to items.


```markdown
paragraph text Lorem ipsum dolor sit amet, consectetur adipisicing elit
{: .some-class}
```

Produces this HTML:
```html
<p class="some-class">paragraph text Lorem ipsum dolor sit amet, consectetur adipisicing elit</p>
```

We could do the same with block quotes like this:

```markdown
> blockquote text sed do eiusmod tempor incididunt ut labore et dolore
{: .special-class}
```

To this HTML:

```html
<blockquote class="special-class">blockquote text sed do eiusmod tempor incididunt ut labore et dolore</blockquote>
```

**Comment by biofilo** [comment]
Sorry about the ambiguity! To be clear, I've been using >s for *inline* quotations—like, when a paragraph from another text appears in one of our articles, intended to be read in a linear fashion as part of our text. You can see how we've used that here:
https://crimethinc.com/2017/08/17/why-we-fought-in-charlottesville-a-letter-from-an-anti-fascist-on-the-dangers-ahead

In addition to that, I'd like to have something else that would make a sentence appear big and bold *outside* the linear text.

IGD has something like this--for example, at https://itsgoingdown.org/cant-stand-rain-report-houston/ the text reading “WE RAN INTO SOME GOOD PEOPLE..." It doesn't look that great, but it's an example of what I mean.

As it stands right now, the only way to make a phrase gigantic and bold is to make it a title, and if I do that, it will mess up the taxonomy of the articles.

Thanks so much!

**Comment by veganstraightedge** [comment]
Do you have examples of how this has been used in the past on the old site? Namely in the features.

https://crimethinc.com/categories/features/

**Comment by veganstraightedge** [comment]
Also, @biofilo. Are you thinking that this would be a big/bold bit of text that also appears somewhere else in the content... truly a pulled quote. Or is this a unique bit of text?

The reason I ask, is that I am of the opinion that pull quotes don't belong on the web. Jeremy Keith sums up my feelings about them very well here.

https://adactio.com/journal/11102

***

If it's not a pull quote, but more of an aside, I'm into that. In fact, I've an issue open for that since before the site launched. It's just not been a priority yet.

https://github.com/crimethinc/website/issues/40

**Comment by biofilo** [comment]
Yeah, not to repeat something, I agree that there is no reason to do that online. Look at https://crimethinc.com/2016/09/28/feature-the-secret-is-to-begin-getting-started-further-resources-frequently-asked-questions ...how the Marge Piercy quote works there. Something like that?

**Comment by veganstraightedge** [comment]
That's great. That's exactly what I was thinking when I created the #40 Aside issue.

It's a slightly pedanical ;)  difference. But one that mattered to me, bc the above opinion about repetition online. Glad we're in agreement on that.

I'm gonna close this issue and link to it from #40. I've got some ideas about how to do it syntactically in Markdown to keep it low friction for authors. Similar to the `[[ ]]` media embeds.
